app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -246,6 +246,7 @@ redhat-lsb-core
 %post --nochroot --log=/mnt/sysimage/root/install.postnochroot.log
 /usr/bin/chvt 3
 echo "Changed to TTY3 for post installation..."
+<%= snippet_if_exists(template_name + " custom postnochroot") %>
```

Comment:
if you add `-` to `%>` it won't even affect the snapshot, it wouldn't be adding extra new line